mendozabree/my-diary-v2
# document endpoints

### Description
This PR will document the current endpoints using [rswag](https://github.com/rswag/rswag).

### Testing PR
To test this:
- Using a local version of this repo
- Fire up the server `rails s`
- Navigate to `http://localhost:3000/api-docs/index.html`
- And there you have it, test it and see how it works.

Screenshots
<img width="1440" alt="Screenshot 2019-11-08 at 13 42 14" src="https://user-images.githubusercontent.com/9901011/68470688-9e400100-022d-11ea-91b7-48436614d07e.png">
